packages/console/src/components/Table/TableEmpty.module.scss

```diff
@@ -0,0 +1,22 @@
+@use '@/scss/underscore' as _;
+
+.tableEmpty {
+  text-align: center;
+
+  .image {
+    text-align: center;
```

Comment:
will text-align inherit from the parent node?

Reply:
ditto in `.title` & `.content`

Reply:
updated